File: SGA_Api/SGA_Api/Models/Traspasos/FinalizarTraspasoArticuloDto.cs
using System.Text.Json.Serialization;

namespace SGA_Api.Models.Traspasos
{
    public class FinalizarTraspasoArticuloDto
    {
        [JsonPropertyName("almacenDestino")]
        public string AlmacenDestino { get; set; }
        
        [JsonPropertyName("ubicacionDestino")]
        public string? UbicacionDestino { get; set; }
        
        [JsonPropertyName("usuarioId")]
        public int UsuarioId { get; set; }

        [JsonPropertyName("ConfirmarAgregarAPalet")]
		public bool? ConfirmarAgregarAPalet { get; set; }
        
        [JsonPropertyName("DejarSuelto")]
		public bool? DejarSuelto { get; set; }
        
        [JsonPropertyName("PaletIdConfirmado")]
		public Guid? PaletIdConfirmado { get; set; }
	}
} 

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/traspasos/FinalizarTraspasoArticuloDto.kt
package com.example.sga.data.dto.traspasos

import com.google.gson.annotations.SerializedName

data class FinalizarTraspasoArticuloDto(
    val almacenDestino: String,
    val ubicacionDestino: String,
    val usuarioId: Int,
    @SerializedName("ConfirmarAgregarAPalet")
    val confirmarAgregarAPalet: Boolean? = null,
    @SerializedName("DejarSuelto")
    val dejarSuelto: Boolean? = null,
    @SerializedName("PaletIdConfirmado")
    val paletIdConfirmado: String? = null  // ID del palet seleccionado mediante escaneo GS1
) 